### ISSUE TITLE
Running coverage report with --only parameter

### ISSUE THREAD

**Issue by dominikbrandon** [badges/shields#7133]
Currently the [documentation](https://github.com/badges/shields/blob/b3199b23b35c3686962e4d18a7f3dd4e887ff40f/doc/service-tests.md#code-coverage) states that in order to run code coverage report for a single service, you need to execute the following:
```sh
npm run coverage:test:services -- --only=wercker
```

However, on my machine, this results in running all tests, effectively ignoring the `--only` parameter.
```sh
➜  shields git:(master) node --version
v14.18.0
➜  shields git:(master) npm --version                                      
7.24.2
➜  shields git:(master) npm run coverage:test:services -- --only=wercker

> shields.io@0.0.0 coverage:test:services
> c8 npm run test:services "--only=wercker"

npm WARN invalid config only="wercker" set in command line options
npm WARN invalid config Must be one of: null, prod, production
```

I've discovered that in order to pass this parameter properly to the underlying npm command, you need to run it as follows (notice the double `--`):
```sh
npm run coverage:test:services -- -- --only=wercker
```

What are your thoughts about that? Does it happen to anyone else, or is it just me? Did it change over time? Anyway, what can we do about that? The simplest solution is to update the docs with the double `--`, but maybe you'd prefer to somehow update the npm scripts in `package.json`? I'll be happy to fix that, but please let me know what is the desired approach.

**Comment by calebcartwright** [comment]
Yes you are correct, thanks for the report. I think once upon a time what's currently documented worked, but for a while now the coverage script needs the double hyphens so that the service test filter arg actually gets passed to the correct place.

Would you be interested in a PR to update the documentation accordingly?

**Comment by dominikbrandon** [comment]
Sure thing 🙂 I'll do it right away.
